### ISSUE TITLE
Trying to add a point to a SequenceFlow may break editing if zoom level != 100%

### ISSUE THREAD

**Issue by var-const** [bpmn-io/bpmn-js#595]
Reproduction steps:
1. Open demo.bpmn.io/new
2. Add a Task to the diagram.
3. Add a Gateway to the diagram. Position it to the right of the Task so that their centers are aligned horizontally (see screenshots).
4. Zoom out several steps.
5. Zoom in step by step, each time clicking on the Gateway and checking if the pad (context menu) appears (it doesn't appear under certain zoom levels, which is another [bug](https://github.com/bpmn-io/bpmn-js/issues/596)). The new zoom level must be smaller than the initial zoom level.
6. From the pad, add a SequenceFlow from the Gateway to the Task. Connect it to the upper-right part of the Task, so that the resulting SequenceFlow has a bend point (see screenshots).
7. Try moving that bend point up.

Result:
After releasing the mouse button, the bend point continues to move with the cursor. Sometimes it is "released" after making several mouse clicks, but after that editing diagram doesn't work correctly and produces weird glitches (see last screenshot). There is an error message in console: "app.min.js:5 Uncaught Error: expected between [1, 2] circle -> line intersections".

I would speculate that this may be a floating point issue, specifically comparing floating point values.

Note 1: the reproduction rate is not 100%, but it's not very hard to reproduce using these steps, either.

Note 2: this bug appears under many circumstances, it's just the most reliable set of reproduction steps that I was able to come up with.

![1](https://cloud.githubusercontent.com/assets/8216808/17004786/ac3fe2b0-4ea4-11e6-8bed-62daf3641d26.png)
![2](https://cloud.githubusercontent.com/assets/8216808/17004789/ac44fb24-4ea4-11e6-943f-0458447be81f.png)
![3](https://cloud.githubusercontent.com/assets/8216808/17004788/ac44762c-4ea4-11e6-9c79-ae35189e50d7.png)
![4](https://cloud.githubusercontent.com/assets/8216808/17004790/ac470de2-4ea4-11e6-8a37-0239f4ded09e.png)
![5](https://cloud.githubusercontent.com/assets/8216808/17004825/e1322d52-4ea4-11e6-9db5-c8e98fb51358.png)

**Comment by var-const** [comment]
I have suspicion about this line:

https://github.com/bpmn-io/bpmn-js/blob/b881ca808696a43b98f2f20d44a81beafe5e92e5/lib/features/modeling/behavior/util/LineAttachmentUtil.js#L192

```javascript
function fenced(n, rangeStart, rangeEnd) {
  return min(rangeStart, rangeEnd) <= n && n <= max(rangeStart, rangeEnd);
}
```

**Comment by nikku** [comment]
Thanks for this bug report. We will look into it and investigate a fix.

**Comment by nikku** [comment]
Seems like this got fixed in bpmn-js `v0.16.1`: https://github.com/bpmn-io/bpmn-js/issues/590.
